front-page.php

```diff
@@ -451,20 +459,38 @@
     $press_pub = $press_meta['_igv_press_publication'][0];
     $press_url = $press_meta['_igv_press_url'][0];
```

Comment:
and yea the code only works if the user fills in the url cause otherwise theres no point in printing anything